src/Form/Type/EntryFilterType.php

```diff
@@ -205,6 +205,24 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'choices' => array_flip($this->repository->findDistinctLanguageByUser($user->getId())),
                 'label' => 'entry.filters.language_label',
             ])
+            ->add('sortOrder', ChoiceFilterType::class, [
+                'apply_filter' => function (QueryInterface $filterQuery, $field, $values) { },
+                'choices' => [
+                    'entry.sort.ascending' => 'asc',
+                    'entry.sort.descending' => 'desc',
+                ],
+                'label' => 'entry.sort.order_label',
+            ])
+            ->add('sortType', ChoiceFilterType::class, [
+                'apply_filter' => function (QueryInterface $filterQuery, $field, $values) { },
+                'choices' => [
+                    'entry.sort.by.creation_date' => 'createdAt',
+                    'entry.sort.by.title' => 'title',
+                    'entry.sort.by.url' => 'url',
+                    'entry.sort.by.reading_time' => 'readingTime',
+                ],
+                'label' => 'entry.sort.status_label',
+            ])
```

Comment:
I wouldn't add those to the form, but instead make them optional GET parameters of the controller, and add links in the UI with the correct values we want.
here the empty apply_filter option is a sign that it's not the way to go 🙂